sdk/storage/storage-queue/src/MessagesClient.ts

```diff
@@ -244,16 +256,15 @@ export class MessagesClient extends StorageClient {
             extractedCreds.accountName!,
             extractedCreds.accountKey
           );
-          urlOrConnectionString = extractedCreds.url + "/" + queueName + "/messages";
+          url = extractedCreds.url + "/" + queueName + "/messages";
```

Comment:
>queueName [](start = 43, length = 9)

due to the if-else here it's unfortunate that we have `queueName` already in this case, but have to parse url to get it

Reply:
yes

Reply:
If this bothers you, I can initialize `this._queueName` here itself and can add an if-else to check if `this._queueName` is already initialized.